src/util/cow.h

```diff
@@ -0,0 +1,235 @@
+/*******************************************************************\
+
+Module: Copy on write class
+
+Author: Reuben Thomas, reuben.thomas@diffblue.com
+
+\*******************************************************************/
+
+#ifndef CPROVER_UTIL_COW_H
+#define CPROVER_UTIL_COW_H
+
+#include "util/small_shared_ptr.h"
+#include "util/invariant.h"
```

Comment:
In keeping with practice elsewhere, could you use `#include "small_shared_ptr.h"` and `#include "invariant.h"`, please?